hw/ip/flash_ctrl/dv/flash_ctrl_base_sim_cfg.hjson

```diff
@@ -90,6 +90,7 @@
     {
```

Comment:
Do we really neeed 300 s timeout?
I know some tests runs more than 1s. Can you explain how you got this number?